src/EnergyPlus/FluidProperties.hh

```diff
@@ -662,6 +662,12 @@ struct FluidPropertiesData : BaseGlobalStruct
     std::array<FluidProperties::cached_tsh, FluidProperties::t_sh_cache_size> cached_t_sh;
 #endif
 
+    void init_state(EnergyPlusData &state) override
```

Comment:
The only non-empty `init_state` function thus far.

Reply:
I noticed that FluidProperties.cc did not change which means these can also be deleted everywhere, as well as         the variable state.dataFluidProps->GetInput.

        if (state.dataFluidProps->GetInput) {
            GetFluidPropertiesData(state);
            state.dataFluidProps->GetInput = false;
        }

Reply:
I'm keeping this pretty minimal for now (strange, I know). I'm going to rework the FluidProperties API and see the effects downstream before propagating this, but I will do that in the following PR.